src/config.c

```diff
@@ -736,15 +781,14 @@ void configSetCommand(client *c) {
     const char *errstr = NULL;
     const char *invalid_arg_name = NULL;
     const char *err_arg_name = NULL;
-    standardConfig **set_configs; /* TODO: make this a dict for better performance */
+    dict *set_configs = dictCreate(&optionSetDictType);
     list *module_configs_apply;
     const char **config_names;
```

Comment:
I don't think we need `config_names` anymore because they will be stored in `set_configs` anyway and we can iterate over this dict to get all names.

Reply:
Sure, also does order in which the values are set matter? If a dictionary is iterated then the order of the configs might be different from the order in the command.

Reply:
The order should not matter.

Reply:
config_names are used in
RedisModuleConfigChangeV1 cc = {.num_changes = config_count, .config_names = config_names}, 
should we keep config_names then?

Reply:
You're right. Lets keep it.